x/leverage/keeper/migrations.go

```diff
@@ -29,7 +29,9 @@ func (m Migrator) MigrateBNB(ctx sdk.Context) (bool, error) {
 	}
 	token, err := m.keeper.GetTokenSettings(ctx, badDenom)
 	if err != nil {
-		return false, err
+		ctx.Logger().Error("leverage migration skipped due to missing token", "err", err.Error())
+		// If it's not a registered token, then we don't need to run this migration
+		return false, nil
```

Comment:
Is there other reason when the error is returned then not found? If yes then let's check the error before nullifying it

Reply:
Since this line skips / aborts the migration and logs it at error level, it's already the same treatment that an unexpected error (unmarshal fail for example) would get